ios/wrappers/utils/AnyCodable.swift

```diff
@@ -0,0 +1,58 @@
+// To convert dictionary to Codable
+struct AnyCodable: Codable {
```

Comment:
We probably dont need it, iOS SDK is already converting map to Codable, we can maybe utilize that?

Reply:
Discussed on call, we'll be removing `AnyCodable` and use `JSON` instead